static/js/lib/queries/topics.js

```diff
@@ -0,0 +1,23 @@
+// @flow
+
+import { topicApiURL } from "../url"
+import { merge } from "ramda"
+import type { CourseTopic } from "../../flow/discussionTypes"
+import { constructIdMap } from "../redux_query"
+import { createSelector } from "reselect"
```

Comment:
code formatting nit: can you remove the blank line below `// @flow` and separate the `type` import from the rest with a blank line?

Reply:
✔️